### PR TITLE
Redesign Solar System Landing Page

### PR DESCRIPTION
**Pull request by ravjot07**
I have redesigned the Solar System landing page to enhance its visual appeal and improve user experience. The changes are:

- Restyled the Page: Updated the overall design with a modern and responsive layout to ensure optimal viewing across all devices.

- Added a Navbar: Introduced a navigation bar at the top of the page to facilitate easy access to different sections of the website.

- Implemented a Facts Section: Added a dynamic Solar System Facts section that displays a random fact each time the page loads, providing engaging and fresh content for visitors.

- Included a Footer Section: Incorporated a footer at the bottom of the page to display consistent site information and enhance the overall structure.

Previous Landing Page
Solar_system

New Landing Page
solar1
solar2

### CODE REVIEW

File: data/images/index.webp
Comment (quozl) [inline data/images/index.webp:1]: We use png and jpeg within Python activities, so this introduction of webp is out of character and needs to be considered carefully.  For instance what release of the embedded WebKit browser began to support it?  Is there a reason to use the format over and above jpeg?

The file is very large compared to the others in data/images.

File: data/index.css [hunk 1]
```diff
@@ -1,56 +1,293 @@
...
+    .fact-text {
+      font-size: 1.1rem;
+    }
+  
+    .fact-text::before,
+    .fact-text::after {
+      display: none;
+    }
+  }
+
```
Comment (quozl) [inline data/index.css:293]: Indentation unusual.  Trailing space without end of line marker.  My terminal prompt shifted right.

```
$ cat index.css
    }
  }
  $ █
```

File: data/index.html [hunk 2]
```diff
@@ -1,29 +1,145 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <title>Solar System</title>
-  <link rel="stylesheet" href="index.css">
+ <meta charset="UTF-8">
+ <title>Solar System Explorer</title>
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
```
Comment (quozl) [inline data/index.html:7]: Links to off-system resources unwelcome; these activities are to function without internet.

File: data/index.html [hunk 3]
```diff
@@ -1,29 +1,145 @@
...
+     <h2>Solar System Fact</h2>
+     <div id="fact-container">
+       <!-- to display random facts -->
+     </div>
+   </section>
+ </main>
+
+
+ <footer>
+   <p>&copy; 2024 Solar System Explorer. All rights reserved.</p>
```
Comment (quozl) [inline data/index.html:74]: We can't accept content that is copyright and without license, which is what "all rights reserved" means.  This is a blocker.  Seek the legal consent of the copyright owner and obtain a license compatible with our project, such as Creative Commons, or remove the content.

File: data/index.html [hunk 4]
```diff
@@ -1,29 +1,145 @@
...
+     factContainer.appendChild(p);
+   }
+
+
+   // Initialize fact on page load
+   window.addEventListener('DOMContentLoaded', displayRandomFact);
+ </script>
 </body>
 </html>
+
```
Comment (quozl) [inline data/index.html:143]: Unnecessary blank lines at end of file.

**Review by quozl — COMMENTED** [review]
Thanks.

I think the design is too busy, and not in keeping with Sugar activities for elementary age children.  I suggest you review the other activities.

The graphic is imaginary.  As a telescope operator working with astrophysicists, I know what the sky looks like, and this graphic misrepresents reality.  Let's not lie to children without making it clear there is a lie.  I'd prefer a simplified graphic.

I'll be closing this pull request due to the "all rights reserved" text, in order to comply with the copyright.

**Comment by quozl** [comment]
Closing due to copyright content without license.  We apologise for any misuse.

**Comment by ravjot07** [comment]
> Closing due to copyright content without license.  We apologise for any misuse.

@quozl sorry for the misunderstanding I just added this line because I have seen it in so many landing pages... So should I work towards redesigning this page with simple design like should I add that facts section or not if I again try to update this landing page

**Comment by quozl** [comment]
It's not a [landing page](https://en.wikipedia.org/wiki/Landing_page).  It is an activity for offline use that introduces concepts relating to the solar system.  Sorry if that was not obvious from the repository context.  Your screenshots were of a web browser, but you must run the activity inside Sugar in order to test properly.

See the [Sugar documentation](https://help.sugarlabs.org/en/) for more about Sugar activities.  I note this activity does not have an entry yet.

If you can't run Sugar, you might be interested in Sugarizer, which has a Planets activity.

You added the line because you saw it somewhere else; that's copying.  It is critical to understand copyright and licensing, in order to protect yourself and us.  Please do some research on those topics before your next contribution.  Do not copy and paste other people's work without attribution and permission.

**Comment by ravjot07** [comment]
> It's not a [landing page](https://en.wikipedia.org/wiki/Landing_page). It is an activity for offline use that introduces concepts relating to the solar system. Sorry if that was not obvious from the repository context. Your screenshots were of a web browser, but you must run the activity inside Sugar in order to test properly.
> 
> See the [Sugar documentation](https://help.sugarlabs.org/en/) for more about Sugar activities. I note this activity does not have an entry yet.
> 
> If you can't run Sugar, you might be interested in Sugarizer, which has a Planets activity.
> 
> You added the line because you saw it somewhere else; that's copying. It is critical to understand copyright and licensing, in order to protect yourself and us. Please do some research on those topics before your next contribution. Do not copy and paste other people's work without attribution and permission.

@quozl sorry sir for misunderstanding i thought of it as landing page and about that copyright thing it was not someones else work i have seen this copyright this in many websites as i mistook for landing page i thought it might be batter to add this copyright line for solar system for this projects of sugar Labs..... sorry again for misunderstanding

**Comment by ravjot07** [comment]
> It's not a [landing page](https://en.wikipedia.org/wiki/Landing_page). It is an activity for offline use that introduces concepts relating to the solar system. Sorry if that was not obvious from the repository context. Your screenshots were of a web browser, but you must run the activity inside Sugar in order to test properly.
> 
> See the [Sugar documentation](https://help.sugarlabs.org/en/) for more about Sugar activities. I note this activity does not have an entry yet.
> 
> If you can't run Sugar, you might be interested in Sugarizer, which has a Planets activity.
> 
> You added the line because you saw it somewhere else; that's copying. It is critical to understand copyright and licensing, in order to protect yourself and us. Please do some research on those topics before your next contribution. Do not copy and paste other people's work without attribution and permission.

As i have seen this activity is not updated in a while, so there could me many thing needed to be updated... can you suggest me some addition to this project for me to work on.....

**Comment by quozl** [comment]
> i have seen this copyright ... many websites

That's interesting and puzzling.  I had searched and not found the copyright in the wording you had used.  I wanted to make sure we had not drawn the attention of a powerful copyright owner by allowing pull requests without a contributor agreement.

> this activity is not updated in a while, so there could me many thing needed to be updated... can you suggest me some addition to this project for me to work on

It was updated in September 2023, and the solar system has not changed substantially in that time.

(Earlier in the year I observed the asteroid 4 Vesta, and it was exactly where it was meant to be.  It was a pleasing experience to have the mathematics and equipment work well.  I hadn't seen 4 Vesta before at such magnification, and it was not circular like the stars we observe for calibration.)

Ideas for things to work on can be found in our GSoC repository;
* https://github.com/sugarlabs/GSoC

Our projects list;
* https://github.com/orgs/sugarlabs/projects?query=is%3Aopen

We had an older more detailed project list with Python and C language porting tasks, but I can't find it in GitHub any longer.  https://github.com/orgs/community/discussions/68518

**Pull request closed without merging**